lib/src/main/java/org/team100/lib/config/NoteDetector.java

```diff
@@ -68,43 +88,46 @@ public Double getY() {
      *         meters
      */
     public Double fieldRelativeY() {
-            return fieldRelativePose2d().getY();
+        switch (Identity.instance) {
+            case BETA_BOT:
+            case COMP_BOT:
+                return fieldRelativePose2d().getY();
+            case BLANK:
+                return 0.0;
+            default:
+                return 0.0;
+        }
     }
 
     /**
      * @return A robot relative translational value of an object in a camera in
      *         meters
      */
     public Translation2d RobotRelativeTranslation2d() {
-            return new Translation2d(getX(), getY());
+        return new Translation2d(getX(), getY());
     }
 
     /**
      * @return A field relative translational value of an object in a camera in
      *         meters
      */
     public Translation2d FieldRelativeTranslation2d() {
-            return new Translation2d(fieldRelativeX(), fieldRelativeY());
+        return new Translation2d(fieldRelativeX(), fieldRelativeY());
     }
 
     /**
      * @return A robot relative Transform2d value of an object in a camera in meters
      *         for translation and radians for rotation
```

Comment:
maybe add a bit to this comment pointing out the rotational value of this pose, which is the bearing.

Reply:
Fixed